src/core/messages.ts

```diff
@@ -285,6 +261,11 @@ export interface Messages extends EmitsDiscontinuities {
    */
   update(message: Message, update: UpdateMessageParams, details?: OperationDetails): Promise<Message>;
 
+  /**
+   * Add a reaction to a message.
+   */
+  react(message: Message, reaction: string): Promise<void>;
```

Comment:
I'd suggest `addReaction`, only because eventually we'll want to remove

Reply:
True, will change. `addReaction` and `removeReaction` are better than `react` and `removeReaction`.